Makefile

```diff
@@ -10,6 +25,23 @@ TAG ?= latest
 # Image URL to use all building/pushing image targets
 IMG ?= $(REGISTRY)/$(BIN)
 
+# find or download golic
+# download golic if necessary
+golic:
+ifeq (, $(shell which golic))
+	@{ \
+	set -e ;\
+	GOLIC_TMP_DIR=$$(mktemp -d) ;\
+	cd $$GOLIC_TMP_DIR ;\
+	go mod init tmp ;\
+	go get github.com/AbsaOSS/golic@v0.1.3 ;\
```

Comment:
hi, there was some fix with space behind `#` in v0.3.1 (https://github.com/AbsaOSS/golic/releases)
could you upgrade to v0.4.0?

Reply:
@kuritka done, changes pushed